app/assets/stylesheets/templates/_portal.scss

```diff
@@ -30,7 +30,6 @@
     .form-question {
       font-size: 1.9rem;
       margin-bottom: 2.5rem;
-      font-weight: normal;
```

Comment:
Out of curiosity, was this line basically doing nothing?

Reply:
this line was making "what's on your mind" label normal weight instead of bold, because that's what had been asked for in the original mock for the portal messages page. In the mocks for the new update, that label is now bolded. The change will affect the current implementation on GYR portal as well as CTC portal. It is more consistent with what labels generally look like in our app, so seems like a good change.